config/schedule.rb

```diff
@@ -11,3 +11,8 @@
 every :sunday, :at => '7:14 am' do
   rake "multicoops:run TASK=foodsoft:create_upcoming_periodic_tasks"
 end
+
+# Finish ended orders
+every 1.minute do
+  rake "multicoops:run TASK=foodsoft:finish_ended_orders"
+end
```

Comment:
Could it be that this task takes more than a minute to run? Would this interfere? Just to check.

Reply:
it's very unlikely that it takes so long (but i'll address this problem with the new time zone aware scheduler anyway later). it it really takes that long finish! will fail, because the order is already finished and nothing happens